### DISCUSSION TITLE
VRtoolkit option selection?

### DISCUSSION THREAD

**Discussion by Bozothenutter** [retroluxfilm/reshade-vrtoolkit#11] [Q&A] [answered]
I have a HMD without button and SteamVR dashboard does not allow the use of a gamepad.
Is there any other way of selecting options?
Editing an .ini or something? (Which one?)

**Comment by retroluxfilm** [comment, marked as answer]
I think a gamepad should work to use the steamvr dashboard. At least xbox controller but not sure what needs to be set up for that.

If you want to change the settings without using the SteamVR Reshade Dashboard menu you can to the following:

1. Use the `CTRL+POS1/HOME` Hotkey to open the Reshade Menu on the VR mirror on your monitor
2. Press the "Force Reload all effects" to make "vrtoolkit.fx" to show up in the list
![grafik](https://user-images.githubusercontent.com/85709777/142743546-aeb66812-5291-4691-8856-58e9be7b92f4.png)
3. Enable vrtoolkit with the checkbox to enable it (but only for the mirror)
4. Select a different preset on the top or disable the performance mode and adjust the modes and settings of the preset to your liking.
5. When done renable performance mode
6. Restart the game to get the changes as well into VR

Editing the files would not work as good because the variable names are only added to the file once they are changed. You can edit them later though in the `ReShade/Presets/` folder

(It is certainly a lot slower to use that method, that using the dashboard menu to directly see the effects in VR.)
